src/static/riot/competitions/detail/_registration.tag

```diff
@@ -0,0 +1,95 @@
+<registration>
+    <div if="{!status}" class="ui grid">
+        <div class="row">
+            <p>You have not yet registered for this competition.</p>
+            <p>
+                To participate in this competition, you must accept its specific <a href="" onclick="{show_modal}">terms and conditions</a>. After you
```

Comment:
Hmm.. if you remove the href does it still need prevent_default and such wiring, I wonder?

Reply:
Removing the href also removes the `hover` styles of the link. Could add those back manually, but i think `preventDefault` is the easier solution to the problem.